Makefile

```diff
@@ -133,12 +135,10 @@ $(LAST_CHECKPOINT): unicharset lists $(PROTO_MODEL)
 	  --traineddata $(PROTO_MODEL) \
           --old_traineddata $(TESSDATA)/$(START_MODEL).traineddata \
 	  --continue_from data/$(START_MODEL)/$(START_MODEL).lstm \
-	  --net_spec "[1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c`head -n1 data/unicharset`]" \
```

Comment:
Why is it superfluous? It was used to fix the number of output classes (i.e. characters).

Reply:
There are 4 different options for LSTM training - see https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract-4.00#tutorial-guide-to-lstmtraining

- Training From Scratch
- Fine Tuning for Impact
- Fine Tuning for ± a few characters
- Training Just a Few Layers

Complete  --net_spec is given only for training from scratch. The other three options do not require that.

Reply:
```
	lstmtraining \
	  --traineddata $(PROTO_MODEL) \
          --old_traineddata $(TESSDATA)/$(START_MODEL).traineddata \
	  --continue_from data/$(START_MODEL)/$(START_MODEL).lstm \
	  --model_output data/checkpoints/$(MODEL_NAME) \
	  --train_listfile data/list.train \
	  --eval_listfile data/list.eval \
	  --max_iterations 400	lstmtraining \
	  --traineddata $(PROTO_MODEL) \
          --old_traineddata $(TESSDATA)/$(START_MODEL).traineddata \
	  --continue_from data/$(START_MODEL)/$(START_MODEL).lstm \
	  --model_output data/checkpoints/$(MODEL_NAME) \
	  --train_listfile data/list.train \
	  --eval_listfile data/list.eval \
	  --max_iterations 400
```

The above is meant to be for the option - https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract-4.00#fine-tuning-for--a-few-characters

 --max_iterations should be 3600 as per Ray's example.